aggregator/deps.ts

```diff
@@ -39,18 +39,18 @@ export type {
   Operation,
   PublicKey,
   Signature,
-} from "https://esm.sh/bls-wallet-clients@0.5.0-d6d7dd4";
+} from "https://esm.sh/bls-wallet-clients@0.5.2";
 
 export {
   Aggregator as AggregatorClient,
   BlsWalletWrapper,
   getConfig,
   VerificationGateway,
   VerificationGateway__factory,
-} from "https://esm.sh/bls-wallet-clients@0.5.0-d6d7dd4";
+} from "https://esm.sh/bls-wallet-clients@0.5.2";
```

Comment:
Is there a reason you published non-experimental 0.5.2?

Reply:
`0.5.0` failed webpack transpilation due to .ts files being included from typechain as you pointed out in https://github.com/jzaki/bls-wallet/pull/75#discussion_r776168168 . I thought I had a fix in `0.5.1` but realized shortly thereafter I had made a mistake. `0.5.2` has the .ts files properly removed.

Given that `0.4.x` -> `0.5.0` broke package level compatibility for the extension and potentially other similar consumers, it seemed prudent to push a prod tag resolving the issue.